app/views/candidates/home/index.html.erb

```diff
@@ -21,9 +21,12 @@
       further information.
     </p>
 
-    <%= link_to 'Start now', new_candidates_school_search_path,
+    <%= link_to new_candidates_school_search_path,
         role:'button',
-        class:'govuk-button govuk-button--start govuk-!-margin-top-2 govuk-!-margin-bottom-8' %>
+        class:'govuk-button govuk-button--start govuk-!-margin-top-2 govuk-!-margin-bottom-8',
+        data: { module: 'govuk-button' } do %>
```

Comment:
I think there's enough duplication throughout the PR we should be adding our own button link helper? We'll then be reducing duplication with this PR?

```ruby
def btn_link_to(*args, **options, &block)
   options[:class] = "govuk-button #{options[:class]}"  
  options[:data] ||= {}
  options[:data][:module] ||= 'govuk-button'

  link_to(*args, **options, &block)
end
```